packages/template-retail-react-app/app/pages/login/index.jsx

```diff
@@ -52,62 +60,103 @@ const Login = ({initialView = LOGIN_VIEW}) => {
-                        const shouldMergeBasket = hasBasketItem && prevAuthType === 'guest'
-                        if (shouldMergeBasket) {
-                            mergeBasket.mutate({
-                                headers: {
-                                    // This is not required since the request has no body
-                                    // but CommerceAPI throws a '419 - Unsupported Media Type' error if this header is removed.
-                                    'Content-Type': 'application/json'
-                                },
-                                parameters: {
-                                    createDestinationBasket: true
-                                }
-                            })
-                        }
                     } catch (error) {
                         const message = /Unauthorized/i.test(error.message)
                             ? formatMessage(LOGIN_ERROR_MESSAGE)
                             : formatMessage(API_ERROR_MESSAGE)
                         form.setError('global', {type: 'manual', message})
                     }
-                } else if (loginType === 'passwordless') {
+                    handleMergeBasket()
+                } else if (loginType === LOGIN_TYPES.PASSWORDLESS) {
                     setCurrentView(EMAIL_VIEW)
                     setPasswordlessLoginEmail(data.email)
-                    // Handle passwordless login logic here
+                    try {
+                        authorizePasswordlessLogin(data.email)
+                    } catch (e) {
+                        form.setError('global', {
+                            type: 'manual',
+                            message: formatMessage(API_ERROR_MESSAGE)
+                        })
+                    }
                 }
             },
-            email: async (data) => {
-                // Handle resend passwordless email logic here
+            email: () => {
+                try {
+                    authorizePasswordlessLogin(passwordlessLoginEmail)
+                } catch (e) {
+                    form.setError('global', {
+                        type: 'manual',
+                        message: formatMessage(API_ERROR_MESSAGE)
+                    })
+                }
             }
         }[currentView](data)
     }
 
-    // If customer is registered push to account page
+    useEffect(() => {
+        if (path === '/passwordless-login-landing' && isSuccessCustomerBaskets) {
+            const token = queryParams.get('token')
+            try {
+                loginWithPasswordlessAccessToken(token)
+            } catch (e) {
+                const message = /Unauthorized/i.test(e.message)
+                    ? formatMessage(INVALID_TOKEN_ERROR_MESSAGE)
+                    : formatMessage(API_ERROR_MESSAGE)
+                form.setError('global', {type: 'manual', message})
+            }
+        }
+    }, [path, isSuccessCustomerBaskets])
```

Comment:
I think this block of code is worthy of a comment on top of it. I'm assuming this the logic that handles when you land on the page with a token from an email or something?
Also, is the `/passwordless-login-landing` something that is configurable? We might want to put that value somewhere that is more visible to be customized.